### PR TITLE
ft-implement notification pane

### PR DESCRIPTION
**Pull request by NNesta**

### CODE REVIEW

File: test/integration/dashboardHeaders.test.js [hunk 1]
```diff
@@ -9,10 +9,11 @@ describe('Fetch All Trip Requests', () => {
...
 
-    fireEvent.click(screen.getByTestId('profile'));
-    expect(screen.getByTestId(/\b(notification-toggle)\b/i)).toBeInTheDocument();
-    await waitFor(() => {
-      expect(screen.getByText(/\b(Sign out)\b/i)).toBeInTheDocument();
-    });
+    // fireEvent.click(screen.getByTestId('profile'));
+    expect(screen.getByTestId(/\b(notification-pane)\b/i)).toBeInTheDocument();
+    fireEvent.click(screen.getByTestId('notification-pane'));
+   /*  await waitFor(() => {
```
Comment (mucyoalexaxel) [inline test/integration/dashboardHeaders.test.js]: @NNesta remove these comments.

**Review by mucyoalexaxel — DISMISSED** [review]
@NNesta kindly address the changes

File: test/integration/dashboardHeaders.test.js (hunk 1 above)
Comment (NNesta) [reply test/integration/dashboardHeaders.test.js]: Done @mucyoalexaxel

**Review by mucyoalexaxel — DISMISSED** [review]
Nice Work @NNesta

**Review by iyaremye-faustin — DISMISSED** [review]
working

**Review by iyaremye-faustin — APPROVED** [review]
Working

**Review by rukundo-kevin — APPROVED** [review]
The notification panel is being displayed as expected. Great work Nesta.

**Pull request merged**